docs/sources/k6/next/testing-guides/performance-testing-grpc-services.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Performance testing gRPC services
+description: Learn the basics of gRPC and how to write a gRPC performance test in k6.
+weight: 350
+---
+
+# Performance testing gRPC services
+
+In this guide, you'll learn the basics of gRPC and how to write a gRPC performance test using k6.
+
+## Before you begin
+
+- [Install k6 v0.29](https://grafana.com/docs/k6/<K6_VERSION>/set-up/install-k6/) or higher
```

Comment:
Do we end lists with a dot?

```suggestion
- [Install k6 v0.29](https://grafana.com/docs/k6/<K6_VERSION>/set-up/install-k6/) or higher.
```

Reply:
Yes, we should! Thanks for catching that.